.github/scripts/test-openjdk-pullrequests.py

```diff
@@ -814,9 +833,11 @@ def get_pr_to_test(untested_prs, visited):
                     # Extract JDK and static-libs bundles
                     try:
                         with zipfile.ZipFile(archive, 'r') as zf:
-                            if not any((zi.filename.startswith("static-libs") for zi in zf.infolist())):
-                                untested_prs.setdefault("they are missing the static-libs bundle (added by JDK-8337265)", []).append(pr)
-                                continue
+                            if not (os.path.isdir("extracted") and os.listdir("extracted")):
```

Comment:
This is testing whether the `extracted` directory is empty as a proxy for static-libs not being downloaded and extracted above right?

Reply:
As far as I can tell, the canary will silently pass on a future OpenJDK PR (after JDK-8350443) that removes the logic for static-libs uploading.

Reply:
by silently pass you mean situation like https://github.com/dougxc/openjdk-pr-canary/actions/runs/13454954377 ?